label_studio/core/utils/common.py

```diff
@@ -657,3 +657,10 @@ def __exit__(self, type_, value, traceback):
                 sender=sender,
                 dispatch_uid=dispatch_uid
             )
+
+
+def get_fields_from_request(request):
+    fields = request.query_params.get('include', None)
```

Comment:
maybe `include` => 'fields` ? because we have a similar field name in api/tasks.

Reply:
`fields` was default for drf-flexfields, but we already had this parameter with different meaning (`fields=all`)